docs/README.md

```diff
@@ -14,7 +14,6 @@ Django/Wagtail combinations as [supported](http://docs.wagtail.io/en/latest/rele
 
 In order to keep for CI build time from growing out of control, not all Python/Django/Wagtail combinations will be tested.
 
-Test as follow:
-- All supported Django/Wagtail combinations with Python `2.7` (since it's the only version in the `2.x` series we support).
+Test as follows:
 - All supported Django/Wagtail combinations with the latest supported Python version of the `3.x` series.
 - The latest supported Django/Wagtail combination for the remaining Python versions.
```

Comment:
```suggestion
- All supported Django/Wagtail combinations with the latest supported Python version of the `4.x` series.
- The latest supported Django/Wagtail combination for the remaining Python versions.
```

Reply:
Updated to `4.x`

Reply:
Added `5.0` for the Wagtail 5.0 upgrade: https://github.com/springload/wagtail-django-recaptcha/pull/42/commits/06e7784708b10f4a29ee4e4cfd4b044e72e57857#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938L18